libraries/GCS_MAVLink/GCS.h

```diff
@@ -354,6 +354,9 @@ class GCS_MAVLINK
     void send_rc_channels() const;
     void send_rc_channels_raw() const;
     void send_raw_imu();
+#if AP_MAVLINK_MSG_HIGHRES_IMU_ENABLED
```

Comment:
```suggestion
```

C++ allows us to declare a function in the header and then never implement it.. so you only need to wrap the implementation in defines, always declare it in the header. I know several functions wrap it here but thats old. Just remove this define. See send_opticalflow() as an example.